### ISSUE TITLE
Initialize repo

### ISSUE THREAD

**Issue by WasabiFan** [kyranstar/Hunt-The-Wumpus#1]
Quick list of things that will need to be done before the repo is ready for use:

  - [x] Add everyone as a collaborator (so that we can push directly)
  - [x] Push some initial content (a `README.md`, to start)
  - [x] Make sure that everyone was registered and added successfully (everyone should receive a notification for this over email)

**Comment by kyranstar** [comment]
I'm still trying to figure out Visual Studio + git... There may be a few weird commits

**Comment by WasabiFan** [comment]
Sounds good. What version of VS are you using?

**Comment by WasabiFan** [comment]
@kyranstar I'd recommend we put the SLN and project files in to a single subfolder; then we can have a root directory tree with folders for things like art designer files and write-ups, if we have any, along with a main code directory.

P.S. Just a warning: it seems to have committed the `bin` folder (which will be re-written on each build) in 49eff2d65a5c17da3298dd23b85e888412bb62b6; the `.gitignore` says that this shouldn't be possible... I really don't know what to think of that :confused:.

**Comment by kyranstar** [comment]
Okay, I agree. I'll redo it in an hour or two. I haven't used command line git in forever :P

**Comment by WasabiFan** [comment]
I completely understand. If you want to, you can use Visual Studio's [built-in Git GUI tool](http://blogs.msdn.com/b/visualstudioalm/archive/2013/01/30/getting-started-with-git-in-visual-studio-and-team-foundation-service.aspx) (supports generic Git) if you want. I find that it works well for VS/C# Git projects. There are also other more fully-functional Git GUIs like GitHub for Windows and SmartGit.

**Comment by kyranstar** [comment]
Thanks, I'll try it! Visual Studio is really not working for me right now, so you can redo the commit or I'll see if I can do it at school tomorrow.

**Comment by WasabiFan** [comment]
I was about to start attempting to create a project when I realized that we might want to wait on creating it until we have fully decided on our graphics technology: Some of them are much easier to use if you create the project file from the start using their specific VS type template. So it might just be best to wait at least a week so that we have a chance to talk it over and come to a consensus on some of the details of how we will build the app.

On a related topic, I do have a few ideas on what technologies we might want to investigate (some of them are cross-platform); but that's for another issue/discussion.

**Comment by nikbyk** [comment]
If we want to do cross-platform, would we maybe be able to get a hold of Xamarin for VS? http://xamarin.com/?utm_content=1544888308&utm_term=xamarin%20visual%20studio&utm_campaign=Xamarin+-+%5bBrand%5d+-+Search&utm_source=&utm_medium=

**Comment by nikbyk** [comment]
Idk how good it would be for a game, though

**Comment by kyranstar** [comment]
It looks like there are some games out for it so they can be made but i feel like with something like XNA there's a lot more information on the web. And plus Imo cross-platform isn't such a big deal for our game

**Comment by WasabiFan** [comment]
The library that I was thinking about using (MonoGame) supports Xamarin as a fully-supported platform, so multi-platform support is definitely possible. MonoGame is a full XNA implementation which means most of the XNA docs apply.

And it shouldn't require Xamarin for VS to test, just if/when our game gets large. We can talk to the MS people for more info on that possibility.

XNA itself has been neglected and there aren't any Microsoft-made alternatives (yet).

Anyway, I can write up a full description about our platform and graphics options in a new issue when I get a chance (probably in an hour).
